guides/common/modules/proc_configuring-podman-to-trust-the-ca.adoc

```diff
@@ -0,0 +1,33 @@
+[id="Configuring_podman_to_trust_the_CA_{context}"]
+= Configuring Podman and Docker to trust the Certificate Authority
+
+Podman uses two paths to locate the CA file, namely, `/etc/containers/certs.d/` and `/etc/docker/certs.d/`.
+
+* You might first need to create the location using:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# mkdir -p /etc/containers/certs.d/hostname.example.com
+----
+* Copy the root CA file to one of these locations, with the exact path determined by the server hostname, either `{project-context}-server` or `{smart-proxy-context}-server`, and naming the file `ca.crt`:
+
+[options="nowrap", subs="+quotes,attributes"]
+----
+# cp rootCA.pem /etc/containers/certs.d/hostname.example.com/ca.crt
+----
+
+* Alternatively, if you are using Docker, you can copy the root CA file to the equivalent Docker directory:
+
+[options="nowrap", subs="+quotes,attributes"]
+----
+# cp rootCA.pem /etc/docker/certs.d/hostname.example.com/ca.crt
+----
+
+You no longer need to use the `--tls-verify=false` option when logging in to the registry:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman login hostname.example.com
```

Comment:
As suggested in https://github.com/theforeman/foreman-documentation/pull/1176#discussion_r833084412, this particular line does not need to run as root.
```suggestion
$ podman login hostname.example.com
```